.buildkite/pipeline.build_core.yml

```diff
@@ -10,5 +10,5 @@
     - ./ci/env/env_info.sh
     - pip install ray[client]
     - bazel test --config=ci $(./ci/run/bazel_export_options) 
-      --test_tag_filters=client_tests,small_size_python_tests,-civ1
-      -- python/ray/tests/...
+      --test_tag_filters=client_tests,small_size_python_tests
+      -- $(bazel query "attr(tags, team:core, tests(python/ray/tests/...))") $(yq -r .flaky_tests[] ci/v2/test/core.tests.yml | sed -e 's/^/-/')
```

Comment:
why do we still have the sed thing?

Reply:
could you use a `- (test1 test2)` or `except (test1 test2)`with one single sub, rather than adding a sub in between each test to exclude?